tests/analysis_new_tests/tests/test.js

```diff
@@ -0,0 +1,159 @@
+import { test } from "node:test";
+import fs from "node:fs/promises";
+import path from "node:path";
+import { glob } from "glob";
+import { spawn, spawnSync } from "node:child_process";
+
+// Get the current ReScript version
+const rescriptVersion = spawnSync("./node_modules/.bin/bsc", ["-v"])
```

Comment:
Nitpick, but we probably have a better way to detect the `bsc.exe`.
Using `cli/bin_path.js`. @cometkim probably knows what to use.

Reply:
```js
import { bsc } from "#dev/process";

const { stdout: version } = await bsc(["-v"]);
```

should work.